CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.2.0] - 2018-06-04
+
+### Added
+
+* Provide metrics object as `prometheus.Collector`, for conventional metric registration.
+* Support non-default/global Prometheus registry.
+* Allow configuring counters with `prometheus.CounterOpts`.
+
+### Changed
+
+* Remove usage of deprecated `grpc.Code()`.
+* Remove usage of deprecated `grpc.Errorf` and replace with `status.Errorf`.
+
+### Removed
+
+n/a
```

Comment:
n/a? What do you mean? That we don't have changelog for below and 1.1.0 or what?

Reply:
if that is the case, can we actually put 1.1 and 1.0 headers with n/a mention (or more verbose version "changelog started from 1.2")

Reply:
n/a stands for not applicable, because there was nothing removed in this release